docs/index.rst

```diff
@@ -0,0 +1,32 @@
+.. Reading List documentation master file, created by
+   sphinx-quickstart on Mon Feb  2 15:08:06 2015.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Reading List's documentation!
+========================================
+
+*Reading list* is a service to keep track of a list of articles to read.
+The state of the reading list is synchronized between devices.
```

Comment:
This is a service attached to Firefox Accounts that enable users to keep their Reading List in sync across all of their devices.

source: https://wiki.mozilla.org/Mobile/FxA_-_Reading_List